src/utils/ImageHelper.cpp

```diff
@@ -0,0 +1,54 @@
+#include "ImageHelper.h"
+#include <cassert>
+
+Bitmap::Bitmap(int width, int height, int channels) 
```

Comment:
why do we have the channels argument here? it's always 4 anyway (RGBA)

Reply:
deleted channels as argument for constructor.